mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -226,14 +289,22 @@ CLASS net/minecraft/unmapped/C_sddaxwyk net/minecraft/item/ItemStack
 		ARG 1 holder
 	METHOD m_xvzvtufo getName ()Lnet/minecraft/unmapped/C_rdaqiwdt;
 	METHOD m_yawcimjo getEntityHolder ()Lnet/minecraft/unmapped/C_astfners;
+	METHOD m_yqbabycc createOptionalCodec (Ljava/lang/String;)Lcom/mojang/serialization/MapCodec;
+		ARG 0 fieldName
+	METHOD m_yqsxbejh transmuteCopy (Lnet/minecraft/unmapped/C_gmbqjnle;I)Lnet/minecraft/unmapped/C_sddaxwyk;
 	METHOD m_yrbtnzbt getRarity ()Lnet/minecraft/unmapped/C_mqmixksm;
 	METHOD m_ywdqmcbk getTooltip (Lnet/minecraft/unmapped/C_jzrpycqo;Lnet/minecraft/unmapped/C_taebrtdw;)Ljava/util/List;
 		ARG 1 player
 		ARG 2 context
+	METHOD m_yzsowcuq (Lnet/minecraft/unmapped/C_cpwnhism;)Lnet/minecraft/unmapped/C_cpwnhism;
+		ARG 1 style
+	METHOD m_zakagwqa set (Lnet/minecraft/unmapped/C_pscqxfcs;Ljava/lang/Object;)Ljava/lang/Object;
 	METHOD m_zpjrlkav onCraftByPlayer (Lnet/minecraft/unmapped/C_cdctfzbn;Lnet/minecraft/unmapped/C_jzrpycqo;I)V
 		ARG 1 world
 		ARG 2 player
 		ARG 3 amount
 	METHOD m_zszkcovn hasEnchantments ()Z
+	METHOD m_zzjqqsgn transmuteCopyIgnoreEmpty (Lnet/minecraft/unmapped/C_gmbqjnle;I)Lnet/minecraft/unmapped/C_sddaxwyk;
```

Comment:
transmute?

Reply:
I'll add javadocs to this method, it's pretty complicated
 there's definitely a better name

Reply:
imagine my disappointment when I stare at these for 5 minutes and realise that the better name I've come up with is just yarn's